refs/wg21.json

```diff
@@ -103281,7 +103281,7 @@
         "title": "Toward Improved Optimization Opportunities in C++0x",
         "href": "https://wg21.link/n1664",
         "authors": [
-            "W. Brown,M. Paterno"
+            "W. Brown, M. Paterno"
```

Comment:
There were a couple of places where I noticed missing spaces after `,`s. I've chosen to put them in, because ... they belong.

Reply:
This should really be split up in two author fields.

Reply:
Are these the canonical forms? Or is this the output of a scraper?